src/components/sessions/add-classes/ClassForm.tsx

```diff
@@ -0,0 +1,234 @@
+                  className={classes.deleteButton}
+                >
+                  <RemoveCircle className={classes.deleteButton} />
+                </IconButton>
+              )}
+            </Box>
+            <Box flex="auto">
+              <Box marginBottom={2}>
+                <Typography variant="h4">
+                  Class {classData.index + 1}
+                </Typography>
+              </Box>
+              <FormRow
+                id="session-name"
+                label="Name"
+                questionType={QuestionType.TEXT}
+                variant={FieldVariant.COMPACT}
+              >
+                <OutlinedInput
+                  autoComplete="new-password" // disable autocomplete
+                  className={classes.input}
+                  fullWidth
+                  id={`classname-${classData.index}`}
+                  placeholder="Class name"
+                  onChange={(e) =>
+                    onUpdateClass(i, { ...classData, name: e.target.value })
+                  }
+                  value={classData.name}
+                />
+              </FormRow>
+              <DaysPicker
+                days={classData.days}
+                onChange={(days) => {
+                  onUpdateClass(i, { ...classData, days });
+                }}
+              />
+              <FormRow
+                id="session-location"
+                label="Location"
+                questionType={QuestionType.TEXT}
+                variant={FieldVariant.COMPACT}
+              >
+                <OutlinedInput
+                  autoComplete="new-password" // disable autocomplete
+                  className={classes.input}
+                  fullWidth
+                  id={`location-${classData.index}`}
+                  placeholder="Location"
+                  onChange={(e) =>
+                    onUpdateClass(i, { ...classData, location: e.target.value })
+                  }
+                  value={classData.location}
+                />
+              </FormRow>
+              <FormRow
+                id="class-facilitator"
+                label="Facilitator"
+                questionType={QuestionType.SELECT}
+                variant={FieldVariant.COMPACT}
+              >
+                <Select
+                  aria-label="Class facilitator"
+                  className={classes.input}
+                  displayEmpty
+                  fullWidth
+                  labelId={`class ${classData.index}`}
+                  onChange={(e) =>
+                    onUpdateClass(i, {
+                      ...classData,
+                      facilitator: e.target.value as number,
+                    })
+                  }
+                  value={classData.facilitator || ""}
+                  variant="outlined"
+                >
+                  <MenuItem value="" className={classes.menuItem}>
+                    None
+                  </MenuItem>
```

Comment:
in the designs, we display label as a placeholder so that users know what they're selecting! you can check out how it's done in `Field.tsx` for an example